CHANGELOG.md

```diff
@@ -8,9 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
It won't let me make a suggestion since you didn't edit the line, but let's fix the Markdown hyperlinks at the bottom of the `README`.

```
[Unreleased]: https://github.com/deepgram-devs/deepgram-rust-sdk/compare/0.6.0...HEAD
[0.6.0]: https://github.com/deepgram-devs/deepgram-rust-sdk/compare/0.5.0...0.6.0
[0.5.0]: https://github.com/deepgram-devs/deepgram-rust-sdk/compare/0.4.0...0.5.0
[0.4.0]: https://github.com/deepgram-devs/deepgram-rust-sdk/compare/0.3.0...0.4.0
[0.3.0]: https://github.com/deepgram-devs/deepgram-rust-sdk/compare/0.2.1...0.3.0
```

Reply:
I moved these up to the header lines, so they don't get forgotten in the future.